mmv1/templates/terraform/examples/vertex_ai_featureonlinestore_with_beta_fields_bigtable.tf.erb

```diff
@@ -0,0 +1,23 @@
+resource "google_vertex_ai_feature_online_store" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+  name     = "<%= ctx[:vars]['name'] %>"
+  labels = {
+    foo = "bar"
+  }
+  region = "us-central1"
+  bigtable {
+    auto_scaling {
+      min_node_count         = 1
+      max_node_count         = 2
+      cpu_utilization_target = 80
+    }
+  }
+  embedding_management {
+    enabled = true
+  }
+  force_destroy = true
```

Comment:
Should probably be removed, this will show up in our docs.

Follow-up question to confirm: when this config is run in our tests, I assume it will still be deleted without `force_destroy` because it is not using any feature views or features. Is that correct, or will there be leftover resources?

Reply:
> Follow

1. Can we have force_destroy = true as set for one of the examples ? This would also mean that we'd be addressing the missing test report that is getting generated for `force_destroy` field and also an example of the usage of the field too. 
2. Yes, It'll still be deleted. I've run the tests manually in my test env and the resources got deleted as expected at the end of the test in both force_destroy = true and false cases.

Reply:
Sure, if you feel it is better to include for this example I think that makes sense, and can always be adjusted later. Thanks for checking that resources get deleted!